CHANGELOG.md

```diff
@@ -4,19 +4,102 @@
+- Make ODS namespace configurable ([#312](https://github.com/opendevstack/ods-jenkins-shared-library/pull/312))
+- Document params to odsPipeline ([#126](https://github.com/opendevstack/ods-jenkins-shared-library/issues/126))
+- quickstarters/pushToRemoteStage should support existing local git repo ([#265](https://github.com/opendevstack/ods-jenkins-shared-library/issues/265))
+- New quickstarter stage initRepoFromODS ([#268](https://github.com/opendevstack/ods-jenkins-shared-library/issues/268))
+- Jenkins shared library should support monorepo - multi build / deploy ([#222](https://github.com/opendevstack/ods-jenkins-shared-library/issues/222))
+- Get last successful build ([#155](https://github.com/opendevstack/ods-jenkins-shared-library/issues/155))
+- Integrate SonarQube with pull requests ([#174](https://github.com/opendevstack/ods-jenkins-shared-library/issues/174))
+- Add quickstarter pipeline ([#230](https://github.com/opendevstack/ods-jenkins-shared-library/pull/230))
+- Support for dynamic injection of /docs folder into Dockerfile and MRO label support ([#229](https://github.com/opendevstack/ods-jenkins-shared-library/issues/229))
+- ods-pipeline should have 2 more stages (start / finish) ([#201](https://github.com/opendevstack/ods-jenkins-shared-library/issues/201))
+- verbosity of standard shared libary stages ([#196](https://github.com/opendevstack/ods-jenkins-shared-library/issues/196))
+- Allow easy configuration of compute resources ([#173](https://github.com/opendevstack/ods-jenkins-shared-library/issues/173))
 
 ### Changed
-- ciSkip check happens on slave node rather than master ([#199](https://github.com/opendevstack/ods-jenkins-shared-library/issues/199))
+- Lazily set OpenShift app domain ([#396](https://github.com/opendevstack/ods-jenkins-shared-library/pull/396))
+- Fix rollout race condition and display events in Jenkins log ([#385](https://github.com/opendevstack/ods-jenkins-shared-library/pull/385))
+- bitbucket status setting in MRO on fail ([#370](https://github.com/opendevstack/ods-jenkins-shared-library/pull/370))
+- Skip shared lib build / deploy if commits did not change ([#103](https://github.com/opendevstack/ods-jenkins-shared-library/issues/103))
+- Ease and document Tailor-based preview deploy ([#340](https://github.com/opendevstack/ods-jenkins-shared-library/pull/340))
+- consistently use logger throughout all the shared lib ([#326](https://github.com/opendevstack/ods-jenkins-shared-library/issues/326))
+- Orchestration Pipeline: Merge released code back into main branch ([#367](https://github.com/opendevstack/ods-jenkins-shared-library/issues/367))
+- Use URLs consistently ([#366](https://github.com/opendevstack/ods-jenkins-shared-library/pull/366))
+- MRO performance on large application ([#348](https://github.com/opendevstack/ods-jenkins-shared-library/issues/348))
+- Remove OWASP support ([#225](https://github.com/opendevstack/ods-jenkins-shared-library/issues/225))
```

Comment:
This can be moved to a `Removed` section, see https://keepachangelog.com/en/1.0.0/